### ISSUE TITLE
Add est. route efficiency by elevation profile

### ISSUE THREAD

**Issue by nickolasclarke** [nickolasclarke/dispatch#41]
Currently we fix the drive train efficiency. We should make more accurate estimates of the route efficiency in both directions of travel based on the elevation profile of the route. 

https://github.com/Dungyichao/Electric-Vehicle-Route-Planning-on-Google-Map-Reinforcement-Learning has simple physical model of how this might work.

**Comment by r-barnes** [comment]
I think our largest source of inaccuracy will be not accounting for acceleration/deceleration at bus stops and traffic controls.

Nonetheless, this is probably a kind of low-hanging fruit. I'd suggest acquiring a global digital elevation model (MERIT 90m might work) and querying this to get the elevations.

**Comment by nickolasclarke** [comment]
yes, this is small potatoes as this point, but I wanted to note it, in the off chance we do ever get around to it.
